lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -303,6 +306,13 @@ class FlutterViewEmbedder {
     return Future<bool>.value(false);
   }
 
+  /// Gets preferred screen orientation that were previously set by [setPreferredOrientation].
+  ///
+  /// Returns a list of DeviceOrientation values.
+  Future<List<dynamic>> getPreferredOrientation() async {
```

Comment:
Looks like this should be in plural form?
```suggestion
  Future<List<dynamic>> getPreferredOrientations() async {
```